numalogic/udfs/tools.py

```diff
@@ -57,6 +60,25 @@ def get_df(
     return df[features].astype(np.float32), df["timestamp"].astype(int).tolist()
 
 
+def _update_info_metric(
+    data: np.ndarray, metric_names: Sequence[str], labels: Sequence[str]
+) -> None:
+    """
+    Utility function is used to update the gauge metric.
+    Args:
+        data: data
+        metric_names: metric name in the payload
+        labels: labels.
+    """
+    metric_mean = np.mean(data.reshape(-1, 1), axis=0)
```

Comment:
Why reshape here?

Reply:
postprocess sends the value after taking a mean and we get it like [a, b] for 2 let's say [falied, degraded]